frontend/src/components/maps/hooks/useActiveFeatureLayer.tsx

```diff
@@ -87,18 +87,35 @@ const useActiveFeatureLayer = ({
    * Note: currently this is limited to finding one parent in the case of a building. in the future, we may need to find/display all matching parcels.
    */
   useDeepCompareEffect(() => {
-    const highlightSelectedProperty = async () => {
+    const highlightSelectedProperty = async (latLng: LatLng) => {
       const parcelLayerData = await parcelsService.findOneWhereContains({
-        lat: selectedProperty?.parcelDetail?.latitude || 0,
-        lng: selectedProperty?.parcelDetail?.longitude || 0,
+        lat: latLng.lat,
+        lng: latLng.lng,
```

Comment:
Any reason to not just doing this here `parcelsService.findOneWhereContains(latLng)`